.github/workflows/test.yml

```diff
@@ -1,11 +1,23 @@
 ---
```

Comment:
It would be nice if we pin GO version in one place, like [here](https://github.com/newrelic/nri-redis/blob/master/.github/workflows/push_pr.yml#L15)